CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-03-15
+
+This release is rolling out to all regions - ETA for conclusion 2021-03-24 for public cloud.
+
+### Announcements
+
+* The Azure Kubernetes Service [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) feature will be retired on June 30th, 2021.
+* Once GA AKS will default to its new [GPU specialized image](https://aka.ms/aks/specialized-gpu-image) as the supported option for GPU-capable agent nodes.
+* Kubernetes version 1.17 will be deprecated in the last week of March 2021.
+* Before k8s 1.20 a bug would allow exec probes to run indefinitely, ignoring any timeoutSeconds configuration value. The previous buggy behavior has been fixed, and timeouts are now enforced. Additionally, this change introduces a new default timeout of 1 second. Please audit all your existing exec probes to make sure that it is appropriate to enforce a 1 second timeout. If not, please provide an explicit timeoutSeconds value that is appropriate for each [exec probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes).
+* CSI Drivers will become default for [Kubernetes versions 1.21+](https://docs.microsoft.com/azure/aks/csi-storage-drivers).
+
+### Release Notes
+
+* Bug Fixes
+  * Fixed an issue with using a managed identity created in a different subscription from the cluster while using pod identity [github](https://github.com/Azure/aad-pod-identity/issues/1003#issuecomment-802574090).
+* Behavioral Changes
+  * Made improvements to Cluster AutoScaler for ignoring terminating pods for scale down
```

Comment:
@marwanad could we get a more detailed description here? Is there a particular timeout, or it just ignores them altogether? Is this good enough for the behavior?

Reply:
Looks good enough to me.

> Made improvements to Cluster AutoScaler for ignoring pods that may be stuck on Terminating due to CRI issues for scale down considerations maybe?